app/Resources/static/themes/_global/index.js

```diff
@@ -29,7 +29,20 @@ $(document).ready(() => {
     });
 
     const x = JSON.parse($('#annotationroutes').html());
-    app.include(annotator.storage.http, x);
+    app.include(annotator.storage.http, $.extend({}, x, {
+      onError(msg, xhr) {
+        if (!Object.prototype.hasOwnProperty.call(xhr, 'responseJSON')) {
+          return;
```

Comment:
Normally, this case should not occur, there is allways responseJSON, I could remove this part else I can indeed display `annotator.notification.banner(errorText, 'error');`. What to put as message? How translate it ?